src/parameters.jl

```diff
@@ -20,18 +20,19 @@ value(x::NamedTuple) = map(value, x)
 value(x::Dict) = Dict(k => value(v) for (k, v) in x)
 
 """
-    positive(val::Real, transform::Bijector=Bijectors.Exp(), ε::Real = 1e-12)
+    positive(val::T, transform::Bijector=Bijectors.Exp(), ε=sqrt(eps(T))) where T<:Real
 
 Returns a `Postive`.
 The `value` of a `Positive` is a `Real` number that is constrained to be positive.
 This is represented in terms of an a `transform` that maps an `unconstrained_value` to the
 positive reals.
 Satisfies `val ≈ transform(unconstrained_value)`
 """
-function positive(val::Real, transform::Bijector=Bijectors.Exp(), ε::Real = 1e-12)
-    if val <= 0
-        throw(ArgumentError("Value, $val, is not positive."))
-    end
+function positive(
+    val::T, transform::Bijector=Bijectors.Exp(), ε=sqrt(eps(T)),
+) where T<:Real
+    val > 0 || throw(ArgumentError("Value ($val) is not positive."))
+    val > ε || throw(ArgumentError("Value ($val) is too small, relative to ε ($ε)."))
```

Comment:
This could be one check/error, but we might want to distinguish them?

Reply:
Personally I like how you wrote this, makes it very easy to read.

Reply:
I agree. This seems like the right way to do it.